### ISSUE TITLE
testsuite fails on alpine linux s390x with segfault

### ISSUE THREAD

**Issue by ncopa** [grobian/html2text#14]
html2text 2.0.0 segfautls during `make check` on alpine linux s390x.

Backtrace:

```
Reading symbols from ../html2text...
[New LWP 17526]
Core was generated by `../html2text -rcfile .html2textrc -from_encoding utf-8 utf-8=unicode-hex-refere'.
Program terminated with signal SIGSEGV, Segmentation fault.
#0  0x000002aa21e155e8 in cmp_nocase (s2=<optimized out>, s1=<optimized out>) at cmp_nocase.h:77
77              return _cmp_nocase(s1, strlen(s1), s2, strlen(s2));
(gdb) bt
#0  0x000002aa21e155e8 in cmp_nocase (s2=<optimized out>, s1=<optimized out>) at cmp_nocase.h:77
#1  get_attribute (as=<optimized out>, name=name@entry=0x2aa21e252f4 "ALIGN", dflt=dflt@entry=-1, 
    s1=s1@entry=0x2aa21e252ee "LEFT", v1=v1@entry=0) at html.cpp:635
#2  0x000002aa21e23138 in create_lcs (column_spacing=1, number_of_columns_return=<synthetic pointer>, 
    number_of_rows_return=<synthetic pointer>, lcs_return=0x3ffda47e048, right_border_width=<optimized out>, 
    left_border_width=<optimized out>, w=79, t=...) at auto_ptr.h:79
#3  Table::format (this=0x3ffbd261740, w=79, halign=2) at table.cpp:498
#4  0x000002aa21e0b4bc in format (elements=0x3ffbd265a70, w=79, halign=halign@entry=0) at format.cpp:1419
#5  0x000002aa21e0b7d8 in Body::format (this=0x3ffbd261500, w=79, halign=<optimized out>) at format.cpp:177
#6  0x000002aa21e231be in create_lcs (column_spacing=1, number_of_columns_return=<synthetic pointer>, 
    number_of_rows_return=<synthetic pointer>, lcs_return=0x3ffda47e3b8, right_border_width=<optimized out>, 
    left_border_width=<optimized out>, w=79, t=...) at table.cpp:229
#7  Table::format (this=0x3ffbd2617a0, w=79, halign=1) at table.cpp:498
#8  0x000002aa21e0b4bc in format (elements=0x3ffbd265060, w=79, halign=halign@entry=1) at format.cpp:1419
#9  0x000002aa21e0b7d8 in Body::format (this=0x3ffbd261ab0, w=79, halign=<optimized out>) at format.cpp:177
#10 0x000002aa21e23dd0 in compute_row_heights (row_spacing=<optimized out>, column_spacing=<optimized out>, 
    column_widths=<optimized out>, row_heights_return=<optimized out>, number_of_rows=<optimized out>, 
    lcs_in_out=<optimized out>) at table.cpp:428
#11 Table::format (this=0x3ffbd255940, w=79, halign=0) at table.cpp:558
#12 0x000002aa21e0d160 in format (os=..., halign=0, w=79, indent_left=0, elements=0x2aa21e35c50)
    at format.cpp:1487
#13 Body::format (this=<optimized out>, indent_left=<optimized out>, w=<optimized out>, 
    halign=halign@entry=0, os=...) at format.cpp:202
#14 0x000002aa21e0d4fc in Document::format (this=0x3ffbd169be0, 
    this@entry=<error reading variable: value has been optimized out>, indent_left=indent_left@entry=0, 
    w=79, halign=halign@entry=0, os=...) at format.cpp:158
#15 0x000002aa21e1ace0 in HTMLDriver::process (this=0x3ffda47ee08, document=...) at HTMLDriver.cpp:65
#16 0x000002aa21e1d366 in html2text::HTMLParser::parse (
    this=<error reading variable: value has been optimized out>)
    at /usr/include/c++/10.2.1/bits/stl_vector.h:1043
#17 0x000002aa21e1b0e0 in HTMLDriver::parse (this=this@entry=0x3ffda47ee08) at HTMLDriver.cpp:41
#18 0x000002aa21e07e7c in main (argc=<optimized out>, argv=<optimized out>) at html2text.cpp:265
```

**Comment by grobian** [comment]
wonder if there's a way to narrow down on this issue somewhat, not familiar with s390 enough to know what' different on this arch to make it segfault

**Comment by grobian** [comment]
this actually is the same issue as #55, and was fixed in 9f912b987e2bce86dd066ebc0f94237ef74bbc15
